CHANGELOG.md

```diff
@@ -2145,6 +2145,9 @@ Other minor changes
   wordsByᵇ     : (A → Bool) → List A → List (List A)
   derunᵇ       : (A → A → Bool) → List A → List A
   deduplicateᵇ : (A → A → Bool) → List A → List A
+  find         : (A → Bool) → List A -> Maybe A
```

Comment:
As you can see in the previous entries in the CHANGELOG, boolean
versions have a name with the `ᵇ` suffix and the others instead use
a `Dec`-like interface.

Reply:
Oh yes, I should also add the corresponding non-`ᵇ` functions. Thanks!